templates/docs/patterns/hero/25-75.md

```diff
@@ -0,0 +1,35 @@
+---
+wrapper_template: '_layouts/segmented-docs.html'
+context:
+  title: Hero | Components
+  segmented_tabs:
+    - title: Medium/Large
+      id: medium-large
+      selected: true
+      example:
+        path: /docs/examples/patterns/hero/hero-signpost
+        before_example_content: |
```

Comment:
This won't scale I'm afraid.
We will potentially need to have a freedom of writing contents of these docs quite freely.

Splitting it into nested metadata will be extremely hard to maintain.